make_ghpages/requirements.txt

```diff
@@ -1,4 +1,2 @@
-jinja2
-# This will need to be updated, pinning it to the first vesion that will
-# support the final v1 specs
-optimade==0.8.1
+jinja2==3.0.0a1
+optimade==0.10.0
```

Comment:
maybe use optimade~==0.10.0 (I think this is the syntax? I.e. accept any 0.10.x version?

Reply:
Put in your suggested change here in your comment, so I could simply add it to a batch together with your other suggested change.
In short, I like your suggestions 👍